install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -10,13 +10,23 @@
 // If you use any setting herein, you forfeit support from Gitpod.
 package experimental
 
-import "k8s.io/apimachinery/pkg/api/resource"
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
 
 // Config contains all experimental configuration.
 type Config struct {
-	Workspace *WorkspaceConfig `json:"workspace,omitempty"`
-	WebApp    *WebAppConfig    `json:"webapp,omitempty"`
-	IDE       *IDEConfig       `json:"ide,omitempty"`
+	Workspace *WorkspaceConfig      `json:"workspace,omitempty"`
+	WebApp    *WebAppConfig         `json:"webapp,omitempty"`
+	IDE       *IDEConfig            `json:"ide,omitempty"`
+	PodConfig map[string]*PodConfig `json:"podConfig,omitempty"`
```

Comment:
:+1: leaving a pointer here bc we might want to discuss whether this is string-indexed or an explicit struct.

Reply:
I think string indexed is nice because every component defines its name via a `Component` constant, like here for the server: 

https://github.com/gitpod-io/gitpod/blob/db07326867158a3ed62c99a3e57310fb1f04e57f/install/installer/pkg/components/server/constants.go#L12